### DISCUSSION TITLE
Avalonia.Skia.Linux.Natives Mismatch

### DISCUSSION THREAD

**Discussion by NullVsNone** [AvaloniaUI/Avalonia#6062] [Q&A]
Following https://github.com/AvaloniaUI/Avalonia/wiki/Run-an-Avalonia-application-on-Raspberry-Pi

After referencing the Skia nuget package, I get 
```
NU1605	Detected package downgrade: SkiaSharp from 2.80.2 to 1.68.0. Reference the package directly from the project to select a different version.	
```

I tried with `Avalonia.Skia` and that also failed to launch. After looking around and finding this discussion: https://github.com/AvaloniaUI/Avalonia/discussions/5654

I realize that it's probably because of outdated documentation? What're the requirements to get Avalonia running on a RP?

**Comment by kekekeks** [comment]
That step is no longer needed, the stock SkiaSharp.NativeAssets.Linux contains an ARM build.

**Reply by kekekeks** [reply]
Just do `dotnet publish -r linux-arm` on your desktop and copy the produced binaries. libunwind8 and libfontconfig1 need to be installed on the device.
